praw/models/reddit/subreddit.py

```diff
@@ -3050,23 +3051,48 @@ def submissions(self, **stream_options):
         return stream_generator(self.subreddit.new, **stream_options)
 
 
-class SubredditStylesheet:
+class SubredditStylesheet(RedditBase):
     """Provides a set of stylesheet functions to a Subreddit.
 
+    **Typical Attributes**
+
+    This table describes attributes that typically belong to objects of this
+    class. Since attributes are dynamically provided (see
+    :ref:`determine-available-attributes-of-an-object`), there is not a
+    guarantee that these attributes will always be present, nor is this list
+    necessarily comprehensive.
+
+    ======================= ===================================================
+    Attribute               Description
+    ======================= ===================================================
+    ``images``              A :class:`list` of images used by the stylesheet.
+    ``stylesheet``          The contents of the stylesheet, as CSS.
+    ======================= ===================================================
+
     For example, to add the css data ``.test{color:blue}`` to the existing
     stylesheet:
 
     .. code-block:: python
 
         subreddit = reddit.subreddit("SUBREDDIT")
-        stylesheet = subreddit.stylesheet()
+        stylesheet = subreddit.stylesheet.stylesheet
```

Comment:
The experience of using this interface is confusing. I would expect that the stylesheet of a subreddit would be `subreddit.stylesheet`. When it's actually `subreddit.stylesheet.stylesheet`, it implies that somehow the subreddit's stylesheet has its own stylesheet.

This confusing interface indicates to me that something about the proposed solution needs to be improved.

I also realize (based on my own testing just now) that this confusing behavior already existed before your changes. The code sample previously was mistaken and should have said

```python
stylesheet = subreddit.stylesheet().stylesheet
```

So you haven't introduced this issue, but if you could come up with a solution to improve this behavior, that would be cool.

Reply:
Maybe property `content` that aliases to this?